modules/aad/domain-services/.bicep/nested_roleAssignments.bicep

```diff
@@ -53,7 +53,7 @@ var builtInRoleNames = {
 }
 
 resource AzureADDS 'Microsoft.AAD/DomainServices@2021-05-01' existing = {
-  name: last(split(resourceId, '/'))!
+  name: last(split(resourceId, '/'))
```

Comment:
Hey @paggarwalmsft, could you kindly provide more info on the error you are experiencing? E.g., is it when deploying locally or through pipelines? Which bicep version are you using to deploy?

Reply:
Echoing @eriquas comment, this character was added to align with a specific null-checker linter rule (see this [PR](https://github.com/Azure/ResourceModules/pull/2806)). That being said, maybe they changed their rules yet again.
Like Erika said, please let us know which Bicep CLI version you're using (`bicep -version`).

Reply:
Bicep CLI version 0.18.4 (1620479ac6)